app/helpers/data_export_checkbox_helper.rb

```diff
@@ -0,0 +1,47 @@
+module DataExportCheckboxHelper
+  def usage_statistics(start_date, end_date)
+    start_date = start_date ? Date.parse(sanitize(start_date)).beginning_of_day : service_start_date
+    end_date = end_date ? Date.parse(sanitize(end_date)).end_of_day : Time.zone.today.end_of_day
+
+    results = {}
+    counts = Hash.new(0)
+
+    responses(start_date, end_date).each do |response|
+      counts[response] += 1
+    end
+
+    counts.map do |(option_text, created_date), count|
```

Comment:
Teensy nitpick (as I'm the one that led you astray): could use an `each` here.

Reply:
👍